src/main.cpp

```diff
@@ -10,16 +7,11 @@ int main(int argc, char** argv)
     UNUSED argc;
     UNUSED argv;
 
-    nautical::math::Vector2<double> vector(4.0f, 3.0f);
-
-    nautical::math::Vector2<double> b(7.0, 4.0);
-
-    vector += b;
+    nautical::Engine *engine = new nautical::Engine();
```

Comment:
Why use `new`, for a simple test like this? Just do a value instead of a pointer.